neofetch

```diff
@@ -2833,9 +2835,16 @@ get_song() {
                     /"title"/ {t=$4} END {print a " \n" b " \n" t}')"
         ;;
 
+        "MellowPlayer"*)
+            song="$(qdbus org.mpris.MediaPlayer2.MellowPlayer3 /org/mpris/MediaPlayer2 \
+                    org.mpris.MediaPlayer2.Player.Metadata |
+                    awk -F':' '/xesam:artist/ {a=$3} /xesam:album/ {b=$3}
+                    /xesam:title/ {t=$3} END {print a " \n" b " \n" t}')"
```

Comment:
Does `get_song_dbus "MellowPlayer3"` work?

Reply:
hah,,, arrgh, you're right.. i think i originally gave up on dbus-send because i was trying "MellowPlayer" instead of "MellowPlayer3" and went to qdbus    :p